osbs/cli/main.py

```diff
@@ -263,6 +263,11 @@ def cmd_cancel_build(args, osbs):
 
 
 def cmd_build(args, osbs):
+    if args.labels:
+        labels = dict(args.labels)
+    else:
+        labels = None
```

Comment:
Nitpick: wouldn't label var have a wrong var scope here? I guess we should set it to None first and then make a dict if ```args.labels``` is set